openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIResponseMapper.java

```diff
@@ -621,6 +623,7 @@ private Optional<ApiResponses> getApiResponsesForArrayTypes(OperationAdaptor ope
             ArraySchema arraySchema = new ArraySchema();
             String type02 = array.memberTypeDesc().kind().toString().trim().split("_")[0].
                     toLowerCase(Locale.ENGLISH);
+            type02 = type02.equals(BYTE) ? BYTE_ARRAY : type02;
```

Comment:
can we simply use `type` for the variable name? :)
```suggestion
            type = type.equals(BYTE) ? BYTE_ARRAY : type;
```

Reply:
same for the `schemas02` variable as well

Reply:
Updated